intake/interface/source/defined_plots.py

```diff
@@ -128,15 +152,99 @@ def selected(self, selected):
         """When plot is selected set, make sure widget stays upto date"""
         self.select.value = selected
 
-    def callback(self, *events):
+    def plot_selected(self, *events):
         for event in events:
             if event.name == 'value':
                 self.pane.object = self._plot_object(event.new)
-            if event.name == 'options':
-                self.instructions.object = self.instructions_contents
+            self.custom.name = "Create" if str(self.select.value) == 'None' else "Edit"
+            self.edit_options.visible = self.custom.name == "Edit"
+
+    def name_changed(self, *events):
+        for event in events:
+            if event.name == "value_input":
+                # Empty name not allowed and name cannot already be in use
```

Comment:
We don't have alerts, bit I wonder if we can inform the user why they cannot save? It could just be carefully covered in docs.

Reply:
I'm adding an alert, so we can give useful feedback to the user.
![Alert screenshot](https://user-images.githubusercontent.com/2807270/204664230-ad5d58ad-d715-43c6-a23d-85ca7c0bcb72.png)